bin/rucio

```diff
@@ -327,36 +356,81 @@ def list_dataset_replicas(args):
     def _append_result(dsn, replica):
         if dsn not in result:
             result[dsn] = {}
-        result[dsn][replica['rse']] = [replica['rse'], replica['available_length'], replica['length']]
-
-    if len(args.dids) == 1:
-        scope, name = get_scope(args.dids[0], client)
-        dmeta = client.get_metadata(scope, name)
-        _fetch_datasets_for_meta(meta=dmeta)
+        if cli_config == 'rich':
+            result[dsn][replica['rse']] = [replica['rse'], str(replica['available_length']), str(replica['length'])]
```

Comment:
Why can't this conversion be done when you're making the table? Seems like an unnecessary check

Reply:
I suppose it could be done in the `generate_table` as well. I just wanted to limit the accepted elements in a row to `rich.console.RenderableType` (similarly to `rich.table.Table.add_rows()`), which does not include integer/float values.